### PR TITLE
R: bugherd.com

### PR DESCRIPTION
**Pull request by ojame**
BugHerd is an ~8 year old SaaS product that allows users to log visual feedback on their websites.

A few months ago, we removed a chunk of code which has not been in use for a long time; setting a third-party cookie when we didn't identify a user.

Cookies are now only set on bugherd.com for authenticated users. If you're an unauthenticated user, you simply make a request to our embedded script (sidebar.js), and depending on if the user has "anonymous feedback" enabled or not, the response is either an empty file or the code to execute to render a "send feedback" tab in the UI. This "anonymous feedback" feature is not something we promote in sales material, and is used on very few of our customer accounts.

Our business model (and code!) is not interested in tracking a user or exposing their privacy. If the above does not meet the requirements to remove the BugHerd domain from the list, please let us know what will.

### CODE REVIEW

**Comment by ryanbr** [comment]
Not trusting the script on this "anonymous feedback", so no Pull.

**Comment by spirillen** [comment]
Just visiting your greapy website add distrust in the domain

<details><summary>Logger output</summary>

| | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +82 |  |  | ce727183-1f79-4316-bbb2-6f14ffb75d8d.moz-extension-scheme | 0,3 | xhr | `https://filters.adtidy.org/extension/ublock/filters/14.txt?_=1` |
| +2 |  |  | bugherd.com | 3 | xhr | `https://cdn.elev.io/sdk/icons/v9/medkit.svg` |
| +2 | /twitter.svg | -- | bugherd.com | 3 | xhr | `https://cdn.elev.io/sdk/icons/v9/twitter.svg` |
| +2 |  |  | bugherd.com | 3 | xhr | `https://cdn.elev.io/sdk/icons/v9/align-left.svg` |
| +2 |  |  | bugherd.com | 3 | xhr | `https://cdn.elev.io/sdk/icons/v9/calendar.svg` |
| +2 |  |  | bugherd.com | 3 | xhr | `https://cdn.elev.io/sdk/icons/v9/question-circle.svg` |
| +2 |  |  | bugherd.com | 3 | xhr | `https://cdn.elev.io/sdk/icons/v9/envelope.svg` |
| +2 | \|\|api.amplitude.com^ | -- | bugherd.com | 3 | xhr | `https://api.amplitude.com/` |
| +2 |  |  | bugherd.com | 3 | xhr | `https://ipa.elev.io/api/settings/566dea6dea15c/en?is_init_load=true&loggedin_only=false&first_name=null&last_name=null&registered_at=null&previewMode=false&url=https%3A%2F%2Fbugherd.com%2F` |
| +2 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/favicon.png` |
| +2 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/webclip.png` |
| +2 | \|\|perfectaudience.com^ | -- | bugherd.com | 3 | script | `https://tag.perfectaudience.com/serve/565d072c46d59aecf20000c0.js` |
| +2 | google-analytics.com/analytics.js | << | bugherd.com |  | script | `https://www.google-analytics.com/analytics.js` |
| +2 | \|\|google-analytics.com^ | -- | bugherd.com | 3 | script | `https://www.google-analytics.com/analytics.js` |
| +2 | @@\|\|amplitude.com/libs/amplitude-$script,3p | ++ | bugherd.com | 3 | script | `https://cdn.amplitude.com/libs/amplitude-5.2.2-min.gz.js` |
| +2 | \|\|googleadservices.com^ | -- | bugherd.com | 3 | script | `https://www.googleadservices.com/pagead/conversion_async.js` |
| +2 | \|\|segment.io^$3p | -- | bugherd.com | 3 | xhr | `https://api.segment.io/v1/p` |
| +1 | no-remote-fonts: * true | -- | bugherd.com | 1 | inline-font | `https://bugherd.com/` |
| +1 |  |  | bugherd.com | 3 | script | `https://cdn.elev.io/sdk/main/v4/1581484906794/elevio-main.js` |
| +1 | no-remote-fonts: * true | -- | bugherd.com | 1 | font | `https://bugherd.com/fonts/Avenir-Medium.woff2` |
| +1 | no-remote-fonts: * true | -- | bugherd.com | 1 | font | `https://bugherd.com/fonts/Avenir-Light.woff2` |
| +1 | no-remote-fonts: * true | -- | bugherd.com | 1 | font | `https://bugherd.com/fonts/Avenir-Heavy.woff2` |
| +1 | no-remote-fonts: * true | -- | bugherd.com | 1 | font | `https://bugherd.com/fonts/Avenir-Book.woff2` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/heart.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/divider.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/quote_1quote.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/laptop-post-its.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/shapes-1.svg` |
| +1 | @@\|\|segment.com/analytics.js/*/analytics.min.js$script | ++ | bugherd.com | 3 | script | `https://cdn.segment.com/analytics.js/v1/btrCF64en7dhdFvgTVvvwlTkeDqwZqzO/analytics.min.js` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/close-icon.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/footer-image.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/bugherd-logo-round.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/Combined-Shape-2_1Combined-Shape-2.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/moz.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/beuch.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/trust-radius.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/g2-crowd.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/capterra.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/star-half.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/chrome-web-store.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/star.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/google.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/mozilla-logo-sml.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/beuch_1.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/3-arrow-up.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/homepage-image.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/culture-amp-logo-sml.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/dan_1dan.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/3-arrow-up-left.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/kanban.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/divider-1-pink.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/melbourne-uni-logo-sml.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/paul.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/3-diamond-horizontal.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/task_details_1.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/divider-2-light-green.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/havas-logo-sml.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/small_1small.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/triangles_downx2.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/report.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/arrow-blue.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/bloomberg-logo.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/isobar-logo.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/mckinsey-logo.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/dyson-logo.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/wunderman-logo.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/amazon-logo.png` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/play-circle.svg` |
| +1 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/BugHerd-text-logo.svg` |
| +1 |  |  | bugherd.com | 3 | script | `https://cdn.elev.io/sdk/bootloader/v4/elevio-bootloader.js?cid=566dea6dea15c` |
| +1 | \|\|googleapis.com^$important | -- | bugherd.com | 3 | script | `https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js` |
| +0 |  |  | behind-the-scene | 0,3 | xhr | `https://255.255.255.255/moz-extension://53cfdd28-130e-46ec-8607-995049ad51d9/?id=472fe2f2ad.3684%2Chttps%3A%2F%2Fbugherd.com%2F&url=https%3A%2F%2Fbugherd.com%2F&top=true&suspend=true` |
| +0 |  |  | bugherd.com | 1 | script | `https://bugherd.com/js/webflow.js` |
| +0 |  |  | bugherd.com | 3 | script | `https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.4.1.min.220afd743d.js?site=5d3a5f5271235b7ae8314c6d` |
| +0 |  |  | bugherd.com | 1 | image | `https://bugherd.com/images/Combined-Shape-2_1Combined-Shape-2.png` |
| +0 | \|\|googleapis.com^$important | -- | bugherd.com | 3 | script | `https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js` |
| +0 |  |  | bugherd.com | 1 | css | `https://bugherd.com/css/bugherd.webflow.css` |
| +0 |  |  | bugherd.com | 1 | css | `https://bugherd.com/css/webflow.css` |
| +0 |  |  | bugherd.com | 1 | css | `https://bugherd.com/css/normalize.css` |
| +0 | no-remote-fonts: * true | -- | bugherd.com | 1 | inline-font | `https://bugherd.com/` |
| +0 |  |  | bugherd.com | 1 | doc | `https://bugherd.com/` |
</details>

image

Non of these referrals have anybodies trust in privacy
